Packs/ProofpointServerProtection/Integrations/ProofpointProtectionServerV2/ProofpointProtectionServerV2.py

```diff
@@ -138,13 +138,14 @@ def test_module(client: Client) -> str:
 
 
 def smart_search(client: Client, args: Dict[str, Any]) -> CommandResults:
-    assert (start_time := parse(args.get('start_time', '24 hours'))), \
+    assert (start_time := parse(args.get('start_time', '24 hours'), settings={'RETURN_AS_TIMEZONE_AWARE': True})), \
         f"Failed parsing start time: {args.get('start_time')}"
-    assert (end_time := parse(args.get('end_time', 'now'))), f"Failed parsing end time: {args.get('end_time')}"
+    assert (end_time := parse(args.get('end_time', 'now'), settings={'RETURN_AS_TIMEZONE_AWARE': True})), \
+        f"Failed parsing end time: {args.get('end_time')}"
     result = client.smart_search_request(
         action=args.get('action'),
-        from_=start_time.isoformat(),
-        to=end_time.isoformat(),
+        from_=start_time.strftime("%Y-%m-%dT%H:%M:%S%z"),
+        to=end_time.strftime("%Y-%m-%dT%H:%M:%S%z"),
```

Comment:
Is `to` rely needed if doing a search to `now`?

Reply:
no real need, removed